lib/middleware/karma.js

```diff
@@ -179,6 +179,10 @@ var createKarmaMiddleware = function (
             var filePath = file.path
             var fileExt = path.extname(filePath)
 
+            if (!files.included.hasOwnProperty(i)) {
+              continue
```

Comment:
this `continue` means the things below are skipped in this case, I don't think we want to do that.